recipe/meta.yaml

```diff
@@ -1,18 +1,18 @@
+{% set name = "diffpy.morph" %}
+{% set version = "0.3.0" %}
 {% set python_min = "3.11" %}
-{%- set name = "diffpy.morph" -%}
-{%- set version = "0.2.0" -%}
 
 package:
-  name: {{ name }}
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.org/packages/source/d/diffpy_morph/diffpy_morph-{{ version }}.tar.gz
-  sha256: fe3741623877907f2d5b002b36c0bb3086493008f9e97a40509023165ab9af38
+  url: https://pypi.org/packages/source/d/diffpy_morph/diffpy_morph-0.3.0.tar.gz
```

Comment:
this seems to have the version hard coded instead of templated.   I think we want to go back to the templated version.